doc/hotfix-process.md

```diff
@@ -0,0 +1,63 @@
+Hotfix Release Process
+======================
+
+Hotfix releases are versioned by incrementing the build number of the latest
+release. For example:
+
+    First hotfix:  v1.0.11   -> v1.0.11-1
+    Second hotfix: v1.0.11-1 -> v1.0.11-2
+
+In the commands below, <RELEASE> and <RELEASE_PREV> are prefixed with a v, ie.
+v1.0.11 (not 1.0.11).
+
+## Create a hotfix branch
+
+Create a hotfix branch from the previous release tag, and push it to the main
+repository:
+
+    $ git branch hotfix-<RELEASE> <RELEASE_PREV>
+    $ git push 'git@github.com:zcash/zcash' hotfix-<RELEASE>
+
+## Merge hotfix PRs
+
+Developer should create hotfix PRs based on the hotfix branch. Each PR should be
+reviewed as normal, and then the following process should be used to merge:
+
+- A CI merge build is manually run, by force-triggering the pr-merge builder
+  with the following fields set:
+
+  - Repository: https://github.com/<DevUser>/zcash
```

Comment:
So the hotfix branch should be pushed to the github of the devuser ? ... but this step isn't mentioned in the 'create a hofix branch' section.

Reply:
This here is not a hotfix branch, it is a PR dev branch *based on* the hotfix branch. The hotfix branch itself is treated as a temporary master.

Reply:
```Create a hotfix branch from the previous release tag, and push it to the main
 +repository:
 
     $ git branch hotfix-<RELEASE> <RELEASE_PREV>
     $ git push 'git@github.com:zcash/zcash' hotfix-<RELEASE>```